src/tree.js

```diff
@@ -192,9 +309,34 @@ var Node = P(function(_) {
 
   _.remove = function() {
     this.jQ.remove();
-    this.postOrder('dispose');
     return this.disown();
   };
+
+  _.isParentSimpleSubscript = function () {
+    if (!this.parent) return false;
+    if (!(this.parent.parent instanceof SupSub)) return false;
+
+    // Mathquill is gross. There are many different paths that
+    // create subscripts and sometimes we don't even construct
+    // true instances of `LatexCmds._`. Another problem is that
+    // the relationship between the sub and the SupSub isn't
+    // completely setup during a paste at the time we check
+    // this. I wanted to use: `this.parent.parent.sub !== this.parent`
+    // but that check doesn't always work. This seems to be the only
+    // check that always works. I'd rather live with this than try
+    // to change the init order of things.
+    if (!this.parent.jQ.hasClass('mq-sub')) return false;
```

Comment:
I'm going to file a follow up issue to document the need for a hacky fix like this. Addressing it is a bigger discussion and would likely require substantial refactoring, but smells like this should at least be documented as areas for improvement that are tracked.

Reply:
https://github.com/mathquill/mathquill/issues/971